README.md

```diff
@@ -1 +1,52 @@
-# webpack-template
\ No newline at end of file
+# ToDo list
+
+> Description of the project.
+
+- In this project, I built a basic website that shows list of things I have to do today
+
+## Built With
+
+- HTML
+- CSS
+- JavaScript
+- Webpack
+
+## Live Demo (if available)
+
+- Not available
+
+### Prerequisites
+
+- Have completed all the previous learning of modules.
+
+## Set up
```

Comment:
- [ ] [OPTIONAL] Kindly state that the user should run `npm i` and then `npm` start to run the project.